spec/factories/services.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :service do
+    name { Faker::Movie.title }
+    category { 0 }
```

Comment:
We reference the enum name instead of the DB integer:

```suggestion
    category { :streaming }
```

Reply:
Check if generator created the category enum with number; if so, fix it as a open source contribuition do FactoryBot